app/scenes/Document/components/Header.js

```diff
@@ -274,4 +282,9 @@ const Status = styled(Action)`
   color: ${(props) => props.theme.slate};
 `;
 
+const TocWrapper = styled(Action)`
+  position: absolute;
+  left: 52px;
```

Comment:
```suggestion
  left: 42px;
```

With the existing padding on the left of those actions this actually needs reducing a bit